public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderTitleItems.tsx

```diff
@@ -19,8 +28,9 @@ export interface Props {
 }
 
 export function PanelHeaderTitleItems(props: Props) {
-  // description
   const { panelDescription, alertState, scopedVars, links, replaceVariables, data, panelId } = props;
+  const styles = useStyles2(getStyles);
+  // description
   const rawDescription = panelDescription || '';
   const descriptionMarkdown = getTemplateSrv().replace(rawDescription, scopedVars);
   const description = renderMarkdown(descriptionMarkdown);
```

Comment:
don't render markdown on every header render, this needs to be lazy (when you hover over it / show tooltip )

Reply:
good catch, I will make the changes :)